mne_bids_pipeline/_config_utils.py

```diff
@@ -55,6 +55,12 @@ def get_fs_subject(
         return f"sub-{subject}"
 
 
+def _has_session_specific_anat(
+    subject: str, session: str | None, subjects_dir: pathlib.Path
+) -> bool:
+    return (subjects_dir / f"sub-{subject}_ses-{session}").exists()
```

Comment:
This is merely moved from `_01_recon_all.py`, as it's now needed in more than one place.